radarr/radarr.go

```diff
@@ -431,3 +431,19 @@ func (r *Radarr) SendCommand(cmd *CommandRequest) (*CommandResponse, error) {
 
 	return &output, nil
 }
+
+// MarkHistoryItemAsFailed marks the given history item as failed by id.
+func (r *Radarr) Fail(historyID int64) error {
+	if historyID < 1 {
+		return fmt.Errorf("invalid history ID: %d", historyID)
```

Comment:
Let's avoid dynamic errors. This triggered a [build error.](https://app.travis-ci.com/github/golift/starr/builds/242611275) 
Recommend:
```golang
fmt.Errorf("%w: invalid history ID: %d", ErrRequestError, historyID)
```

https://github.com/golift/starr/blob/a5e9b3f34ec7015a769e4b95fd4c5254a13230c2/radarr/radarr.go#L254-L255

However, see my next comment about this problem in Sonarr, it requires a slight change here too.